### DISCUSSION TITLE
Vue 3.3

### DISCUSSION THREAD

**Discussion by JensErikChristiansen** [nuxt/nuxt#20823] [Questions] [answered]
Hi! Since Vue 3.3 has come out, what's the best way to upgrade? Manually upgrade my vue dependency (yarn upgrade vue@3.3.x), or can I upgrade it via nuxi, or...?

Thanks in advance.

**Comment by kaikaibenkai** [comment]
Nuxt will soon support Vue 3.3
(see [this comment](https://github.com/nuxt/nuxt/discussions/20823#discussioncomment-5891568))

**Comment by warflash** [comment, marked as answer]
This is the PR that contains the update to 3.3: https://github.com/nuxt/nuxt/pull/20586
